tasks/system_probe.py

```diff
@@ -1527,27 +1527,33 @@ def save_test_dockers(ctx, output_dir, arch, windows=is_windows):
 @task
 def test_microvms(
     ctx,
-    security_groups=None,
-    subnets=None,
+    infra_env,
     instance_type_x86=None,
     instance_type_arm=None,
     x86_ami_id=None,
     arm_ami_id=None,
     destroy=False,
     upload_dependencies=False,
+    ssh_key_name=None,
+    ssh_key_path=None,
+    dependencies_dir=None,
+    shutdown_period=320,
```

Comment:
nit: this isn't a period, since it isn't periodic. It is more of a delay. I also think we shouldn't leave this as seemingly unitless (I realize it is minutes, but that isn't communicated here). I tried to change it to `time.Duration`, but that also involved changing the test definition. Let's fix this in a future PR.